addons/canal/canal_v3.19.yaml

```diff
@@ -3591,7 +3591,7 @@ spec:
         # This container installs the CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: '{{ Registry "docker.io" }}/calico/cni:v3.19.1'
+          image: '{{ Registry "quay.io" }}/calico/cni:v3.19.1'
```

Comment:
`v3.19.1` (as well as other 3.19 versions) is missing multi-arch manifest (so would not work on ARM nodes).
Let's keep it on dokerhub.